scripts/test/SANS/common/general_functions_test.py

```diff
@@ -1,18 +1,25 @@
 from __future__ import (absolute_import, division, print_function)
 import unittest
+import sys
 from mantid.kernel import (V3D, Quat)
 from mantid.api import AnalysisDataService
 from sans.common.general_functions import (quaternion_to_angle_and_axis, create_unmanaged_algorithm, add_to_sample_log,
                                            get_standard_output_workspace_name, sanitise_instrument_name,
                                            get_reduced_can_workspace_from_ads, write_hash_into_reduced_can_workspace,
                                            convert_instrument_and_detector_type_to_bank_name,
                                            convert_bank_name_to_detector_type_isis,
-                                           get_facility, parse_diagnostic_settings, get_transmission_output_name)
+                                           get_facility, parse_diagnostic_settings, get_transmission_output_name, get_output_name)
 from sans.common.constants import (SANS2D, LOQ, LARMOR)
 from sans.common.enums import (ISISReductionMode, ReductionDimensionality, OutputParts,
-                               SANSInstrument, DetectorType, SANSFacility)
+                               SANSInstrument, DetectorType, SANSFacility, DataType)
 from sans.test_helper.test_director import TestDirector
 from sans.state.data import StateData
+from sans.state.save import get_save_builder
+
+if sys.version_info.major > 2:
+    from unittest import mock
```

Comment:
Can you point me to where this import is used? Mocking seems to be a creat tool here.

Reply:
Looks like I added the import but the never used it in this case. I'll remove it now.